### ISSUE TITLE
Download page needs significant changes

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/backend#136]
- [x] No download box present on homepage anymore
- [x] Text on download page is confusing

**Comment by JosephMcArthur** [comment]
So, from my understanding of how this works I'd suggest the following. 

- [ ] Make a new page "downloading". Functionally this is similar to what we have now.
- [ ] Get some simple copy together on how to sign up. All copy can be specialized for this (e.g right now the CTA on the button is "authenticate" which could be clearer...)
- [ ] Hitting download there would link to the onboarding workflow described in #137 (account set up, download, instructions, account page)
- [ ] The download page can be hidden if a user is signed in. 

Now I have a vague idea what to do here, assigning @markmacgillivray

**Comment by markmacgillivray** [comment]
CTA button changed to "Sign in". Will go live soon. Just now the "Download" link goes to the login page, and on that page it does say "sign in / register / download" and under that it says that once you sign in you can get your button. There already is the big "Download" button at the top right of the nav for users who are not signed in, and it should disappear for users that are signed in (fixing that now). If you have simple copy you want added to the login page, just let me know and I can add it. However, it really does seem pretty simple - provide email address, press "sign in". After, get button. The workflow after signing in, on issue #137, has also been commented on by me just now.

**Comment by markmacgillivray** [comment]
I think this one can be closed now, with improvements to the account page giving a workflow, and the authenticate button now saying sign in. We can still add more blurb to the starting "login" page, but for now it says that they should login, and can then get their button, so is fine as the starting point I think. I will close this issue now, and later if new words are wanted on the login page they can be easily added at that time.

**Comment by JosephMcArthur** [comment]
@chealsye can you put together some copy for the page login/download page? Here is what I'd suggest. 

"We use a password-less login system, just provide your account email address and you will receive an email to confirm you are who you say you are." and "Just provide your email address and we will send you a validation link. Then once you are logged in for the first time we will ask you for a few other details." type information should be seen only AFTER they put in the email. We can also shorten that down I think.

An example of how Slack treat this type of system: <img width="589" alt="screenshot 2016-07-15 08 45 10" src="https://cloud.githubusercontent.com/assets/4953703/16867233/977839d0-4a68-11e6-89a3-596ad667514c.png">

<img width="562" alt="screenshot 2016-07-15 08 47 21" src="https://cloud.githubusercontent.com/assets/4953703/16867274/da0e7aca-4a68-11e6-953d-a1e850504937.png">

With "( Your e-mail address will be used as your account username until you choose a username of your own, but otherwise it will NOT be publicised or shared with third parties. )" I think we can move that to later in the sign-up process. Especially if Mark has made a clearer onboarding process. 

Finally, for the call to action on the page we need to condense "Once you sign up and sign in, you will be able to download your Button for your preferred browser." and "No account yet? Sign up now! Just put your email address above and click "Sign in" - we will create a new account for you."  and "Do you already have an account? Sign in now!". This text should go above the email box. For what it's worth, this would probably be even easier if we just had two boxes on the page. One email box for sign in, one for download. Then we could split out copy and a header (download / login / sign in) for each. While, clearly, technically this isn't required, I think from what people normally expect (signing in looks different to login) it would make sense. 

What that should leave us with is a much cleaner page that gives the user the info they need when they need it, rather than have it all on the page at once below where we want them to look.  

**Don't want to close until this is sorted and tested.** 

Finally, @markmacgillivray can we get a download section on the bottom of the homepage again? Like ye'old days? Happy for that to be a separate issue in another milestone if you'd like. Your call on that. 

Other small changes:
- [ ] Remove "Open Access Button and Open Data Button" text from lhs. Seems confusing and doesn't add anything. 
- [ ] Move "questions, contact us" directly under the sign in button.
- [ ] Make sign in button orange (the action color)

**Comment by markmacgillivray** [comment]
Getting lots of little icons in an oab style would help with the content
too - just like the examples you shared. Who/how to proceed with that?

**Comment by JosephMcArthur** [comment]
What types of Icons @markmacgillivray, ones like in the Slack image? This feels like it could be a new issue, but I'm more than happy to get them made for us. 

P.s this is the magic email slack users get.

<img width="842" alt="screenshot 2016-07-15 09 06 02" src="https://cloud.githubusercontent.com/assets/4953703/16867765/9797993a-4a6b-11e6-8d6f-f60578c01bd8.png">

 Can I suggest:

- [ ] Use nice HTML button for magical sign in link in email (similar to as discussed for requests)

**Comment by markmacgillivray** [comment]
Yeah, they're not urgent, but just more graphic representations to support
the text explanations. Useful across the site, and in the emails we send
out.

HTML templating for emails hasn't been done at all yet but should be.

We have the little hands / fists graphics, more things like that is what i
mean, and yeah like in the slack email etc.

How were we going to do site graphics stuff? Are you getting someone in?
Should I get someone? I will be improving layouts and workflows etc, but
could do with a graphic person to make logos, workflow charts for people to
follow the text in various places, etc.

**Comment by JosephMcArthur** [comment]
We have cash to support this, so can get someone in. Stick this in another issue (along with some first ideas on what you'd like) and I'll go via my folks first (as I assume they'll be cheaper) but if that doesn't work we'll move onto yours. If there is other things like this you'd like, just say, I'll source it.

**Comment by markmacgillivray** [comment]
See #137 for latest state of this.

**Comment by JosephMcArthur** [comment]
Copy from Chealsye which I still needt o review and edit: https://docs.google.com/document/d/1JmFsUslUKKlutU0gSqFdUXtAz3-M2j8cdVaRbxQwMjc/edit?usp=sharing

**Comment by JosephMcArthur** [comment]
Okay, have reviewed the text and made edits. Mark, you should feel free to get these up whenever. I'd like one more glance before we push live. 

I'm still not 100% sold on the login and download pages being the same. But this a step in the right direction.

**Comment by markmacgillivray** [comment]
Ah, there is new text here, I have made the new login/download process live, so this is now working, but will add this text in. Will move this issue into current milestone

**Comment by JosephMcArthur** [comment]
Did you add in the text Mark?

**Comment by markmacgillivray** [comment]
The new account / signup / signin page is now quite different, I have made all sections of it viewable right now and informed Chealsye, it is available at /account on the dev site. If any of this is still relevant (perhaps the email content part) it can be added, so keeping this issue open but waiting for further feedback.

**Comment by chealsye** [comment]
Copy done - @markmacgillivray, can you change the download login/signup pages to the following?

[ ] @JosephMcArthur, can you look over this before Mark changes the pages? I know this is super reduced from what we've had - but I don't think we need the extra bits. I'd probably do it differently if the login/download page wasn't the same, but I think this simple approach works for both. 

Download page copy is [here.](https://docs.google.com/document/d/1JmFsUslUKKlutU0gSqFdUXtAz3-M2j8cdVaRbxQwMjc/edit#)


![signin](https://cloud.githubusercontent.com/assets/9299248/19209233/f43cdb32-8cd3-11e6-8f48-350128387bf0.png)

![confirmaccount](https://cloud.githubusercontent.com/assets/9299248/19209234/f43ef818-8cd3-11e6-87b1-47c2ce9658d4.png)

**Comment by JosephMcArthur** [comment]
Looks good, like the more minimal approach. All I'd consider it having "Sign in" or "Sign up" depending on if you click login or download.

**Comment by JosephMcArthur** [comment]
The only thing I might suggest adding to the version for download is a sentence under the "sign up" button that mentions they'll then be able to use the button.

**Comment by chealsye** [comment]
Is it actually possible to have the page differ if they are logging or downloading? Both links going to the same page, so I thought it was a one sized fits all page.

**Comment by markmacgillivray** [comment]
The simpler style with less info here is good, still not sure why to make
things half width if we have nothing else to use the space for, but apart
from that, it is tidy. However one of the reasons we have so much more text
at the moment is because the previous version without so much explanation
was considered not informative enough. Hopefully now with better
functionality and neater layout that will not matter so much.

**Comment by chealsye** [comment]
I commented about the width on the registration details issue.

This is the copy I want to go forward with. I agree with Joe about adding a second sentence in. I'd like to also link to the directions. I'll get you the 1-2 sentence addition by tomorrow evening. However, I don't believe there is a different version for the download/sign up page. Both the sign up and login page is the exact same, so I want it to work or both and be clean without an overabundance of text.

**Comment by markmacgillivray** [comment]
Can make this live then close this issue once DNS problems today are resolved

**Comment by JosephMcArthur** [comment]
Copy changes here are good & important, design changes can wait for later issues.

**Comment by JosephMcArthur** [comment]
Just seen this live, looks good. Made a change to the line order but I think we're good to close here.
